File: problems/003-nth-fibonacci/csharp/Practice.cs
using System.Collections.Generic;
using System.Text.Json;
class Practice : ISolution {
    public object Solve(Dictionary<string, object> c) {
        long n = ((JsonElement)c["n"]).GetInt64();
        // Return the nth Fibonacci number: fib(0)=0, fib(1)=1, fib(2)=1, ...
        return 0;
    }
}

File: problems/003-nth-fibonacci/rust/practice.rs
use crate::json::JVal;

pub fn solve(c: &JVal) -> JVal {
    let _n = c.get("n").as_num() as i64;
    // Return the nth Fibonacci number: fib(0)=0, fib(1)=1, fib(2)=1, ...
    JVal::Num(0.0)
}
